### ISSUE TITLE
Build Failed(react-native 0.35)

### ISSUE THREAD

**Issue by Richard-Cao** [weflex/react-native-wechat#120]
```
react-native-wechat/ios/RCTWeChat.m:292:30: No visible @interface for 'RCTImageLoader' declares the selector 'loadImageWithURLRequest:size:scale:clipped:resizeMode:progressBlock:completionBlock:'
```

**Comment by xiaoneng** [comment]
RCTImageLoader loadImageWithURLRequest 变了，加了新的parameter partialLoadBlock。

在： RCTWeChat.m文件里line 292

加：
partialLoadBlock:nil

原来：
[_bridge.imageLoader loadImageWithURLRequest:imageRequest size:CGSizeMake(100, 100) scale:1 clipped:FALSE resizeMode:RCTResizeModeStretch progressBlock:nil completionBlock:^(NSError *error, UIImage *image) {

新：

 [_bridge.imageLoader loadImageWithURLRequest:imageRequest size:CGSizeMake(100, 100) scale:1 clipped:FALSE resizeMode:RCTResizeModeStretch progressBlock:nil **partialLoadBlock:nil** completionBlock:^(NSError *error, UIImage *image) {

**Comment by yorkie** [comment]
Published at v1.7.0, thank you guys :)
